src/Ocis.php

```diff
@@ -752,14 +776,14 @@ public function createEducationUser(
         ?string $issuerAssignedId = null,
         ?EducationUserApi $apiInstance = null,
     ): EducationUser {
-        if (!isset($this->educationGuzzle)) {
+        if ($this->educationAccessToken === null) {
```

Comment:
make a function similar to `checkIfAccessTokenExists`, or even reuse `checkIfAccessTokenExists`